core/server/models/member-payment-event.js

```diff
@@ -0,0 +1,47 @@
+const errors = require('@tryghost/errors');
+const ghostBookshelf = require('./base');
+
+const MemberPaymentEvent = ghostBookshelf.Model.extend({
```

Comment:
Doesn't this model use the `members_payment_events` table [here](https://github.com/TryGhost/Ghost/blob/main/core/server/data/schema/schema.js#L358)? Also `.select(knex.raw('SUM(volume) as gross_volume'))` should be `.select(knex.raw('SUM(amount) as gross_volume'))`

Reply:
THANKS